### PR TITLE
(PC-9141) Notify user prefilter has been modified

### PR DESCRIPTION
**Pull request by schable**

### CODE REVIEW

File: src/components/pages/Bookings/BookingsRecapTable/Filters/Filters.scss [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    .fw-offer-date {
+      button {
+        outline: none;
+      }
+
+      .field-date {
+        border: 1px solid $grey-medium;
+        box-shadow: 2px 2px 6px 0 $black-shadow;
+        margin-top: rem(12px);
+        padding: 0 1px;
```
Comment (asagot-ansa) [inline src/components/pages/Bookings/BookingsRecapTable/Filters/Filters.scss:131]: rem

File: src/components/pages/Bookings/PreFilters/PreFilters.scss [hunk 2]
```diff
@@ -20,6 +20,11 @@
         border-radius: rem(20px);
         display: flex;
         height: rem(40px);
+        border: 1px solid $grey-medium;
+        box-shadow: 2px 2px 6px 0 $black-shadow;
+        margin-top: rem(12px);
+        padding: 0 1px;
```
Comment (asagot-ansa) [inline src/components/pages/Bookings/PreFilters/PreFilters.scss:26]: rem

File: src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx [hunk 3]
```diff
@@ -571,4 +571,78 @@ describe('components | BookingsRecap | Pro user', () => {
     )
     expect(informationalMessage).not.toBeInTheDocument()
   })
+
+  it('should inform the user that the filters have been modified when at least one of them did and before clicking on the "Afficher" button', async () => {
```
Comment (asagot-ansa) [inline src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx:575]:
```suggestion
  it('should inform the user that the filters have been modified when at least one of them was and before clicking on the "Afficher" button', async () => {
```

File: src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx [hunk 4]
```diff
@@ -571,4 +571,78 @@ describe('components | BookingsRecap | Pro user', () => {
...
+    // GIVEN
```
Comment (asagot-ansa) [inline src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx:576]: Given // When // Then

**Review by asagot-ansa — APPROVED** [review]

File: src/components/pages/Bookings/BookingsRecapTable/Filters/Filters.scss [hunk 5]
```diff
@@ -0,0 +1,263 @@
+.vertical-bar {
```
[the remark this answers is not in the document]
Comment (gvannest) [reply src/components/pages/Bookings/BookingsRecapTable/Filters/Filters.scss:1]: 👍

File: src/components/pages/Bookings/BookingsRecapTable/Filters/Filters.scss (hunk 1 above)
Comment (gvannest) [reply src/components/pages/Bookings/BookingsRecapTable/Filters/Filters.scss:131]: 👍

File: src/components/pages/Bookings/PreFilters/PreFilters.scss (hunk 2 above)
Comment (gvannest) [reply src/components/pages/Bookings/PreFilters/PreFilters.scss:26]: 👍

File: src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx (hunk 3 above)
Comment (gvannest) [reply src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx:575]: 👍

File: src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx (hunk 4 above)
Comment (gvannest) [reply src/components/pages/Bookings/__specs__/BookingsRecap.spec.jsx:576]: 👍

**Pull request merged**